### DISCUSSION TITLE
An alternative way to call DARs except LR

### DISCUSSION THREAD

**Discussion by xiaowenchenjax** [stuart-lab/signac#578] [Q&A] [answered]
Hi Tim,
Based on the integrated scATAC-seq assay, I used the LR method to identify the differentially accessible regions (DAR) between different ages,

Unfortunately, I only got 21 DARs.

Could you recommend me some other methods? Can I try all the other methods in FindMarkers() function?

Thank you.

**Comment by timoast** [comment, marked as answer]
Hi @xiaowenchenjax , we do not recommend using the integrated data for differential expression or differential accessibility. You should use the original, uncorrected assay instead.

**Comment by xiaowenchenjax** [comment]
Hi Tim ,
1.Since I have six libraries of snATAC-seq with different ages. After identifing the cell types. i would like to identify differential accessibility in one cell type comparing young with old, if I did not integrate 6 libraries, (1) they did not have common peaks; (2) I couldn't input two seraut object into FindMarkers() function. 
2.what is the original, uncorrected assay? after integrating, assay peaks?
3. In which case, we need to use integrated data? visualization?
Thanks

**Reply by timoast** [reply]
You need to ensure that the same set of peaks are quantified in each dataset, and merge the datasets together: https://satijalab.org/signac/articles/merging.html

The integrated data should be used for creating low-dimensional representations (tSNE or UMAP) and clustering cells.
